src/ot_orchestration/dags/config/platform.py

```diff
@@ -0,0 +1,131 @@
+"""Configuration class for the platform pipeline."""
```

Comment:
Are you sure? This python file is where the config for the platform dag comes, effectively. More than parsing it is building it. Besides, I don't think we will have more files like this, so that folder would have only one file.

But if you prefer so, I'm fine with that too.

Reply:
Like I mentioned, just for the convention, I will not make a big noise around if you prefer it to stay here :)